src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,7 +1,7 @@
 package edu.kis.vh.nursery;
 
-import edu.kis.vh.nursery.list.IntLinkedList;
-import edu.kis.vh.nursery.list.StackInterface;
+import edu.kis.vh.nursery.stack.IntLinkedList;
+import edu.kis.vh.nursery.stack.StackInterface;
 
 public class DefaultCountingOutRhymer {
 

```

Comment:
intLinkedList  -  wrong name (type StackInterface does not limit the implementation)